mscore/editstafftype.ui

```diff
@@ -683,6 +683,13 @@
                    </widget>
                   </item>
                   <item>
+                   <widget class="QCheckBox" name="showTabFingering">
+                    <property name="text">
+                     <string>Show fingering in tablature</string>
```

Comment:
As mentioned eleswhere, no need for the 'in tablature' as this Option apreqs in tablature staff proerties only anyway.
and the "Show" should probably better be "Allow" or "Use"